sdk/provisioning/Azure.Provisioning/src/Expressions/BicepStringBuilder.cs

```diff
@@ -53,6 +55,13 @@ public BicepStringBuilder Append(BicepInterpolatedStringHandler handler)
         return this;
     }
 
+    public BicepStringBuilder Append(FormattableString value)
```

Comment:
I thought the pr description mentioned this overload is not needed?  Should this still be here?

Reply:
yep. let me remove this one.